lib/generators/rubyblok/migration_generator.rb

```diff
@@ -0,0 +1,57 @@
+require "rails/generators"
+require "rails/generators/active_record"
+
+module Rubyblok
+  module Generators
+    class MigrationGenerator < ::Rails::Generators::NamedBase
+      include ::Rails::Generators::Migration
+      source_root File.expand_path("../templates", __dir__)
+      desc "Installs Rubyblok migration and model files."
+
+      def install
+        if table_exist?
+          migration_template("migration_update.rb.erb", "db/migrate/update_rubyblok_#{plural_file_name}.rb", migration_version:)
+        else
+          migration_template("migration_create.rb.erb", "db/migrate/create_rubyblok_#{plural_file_name}.rb", migration_version:)
+        end
+
+        create_or_update_model
```

Comment:
we should move `add_mixins_to_existing_model` and `generate_new_model_from_template` to `if` statement above

Reply:
```
      def create_or_update_model
        if table_exist?
          add_mixins_to_existing_model
        else
          generate_new_model_from_template
        end
      end
```